interface/modules/custom_modules/oe-module-comlink-telehealth/templates/comlink/appointment/add_edit_event.js.twig

```diff
@@ -0,0 +1,11 @@
+(function(window, comlink) {
+
+    // call our init function with our variables
+    let teleHealthProviders = {{ providers|json_encode|text }};
+    let telehealthCategories = {{ categories|json_encode|text }};
```

Comment:
drop text from above 2 lines (btw, just one more  minor escaping issue for me to look into; then can take a need break from escaping :) )

Reply:
Fixed this.